flake.nix

```
@@ -37,23 +37,27 @@
           overlays = [
             (final: _prev: { fenix = self.inputs.fenix.packages.${final.system}; })
             (import ./overlays/nixpkgs.nix)
+            (import ./overlays/contrast.nix)
           ];
           config.allowUnfree = true;
           config.nvidia.acceptLicense = true;
         };
-        inherit (pkgs) lib;
-        treefmtEval = treefmt-nix.lib.evalModule (pkgs // ourPkgs) ./treefmt.nix;
-        ourPkgs = import ./packages { inherit pkgs lib; };
+        treefmtEval = treefmt-nix.lib.evalModule pkgs ./treefmt.nix;
       in
 
       {
-        devShells = pkgs.callPackages ./dev-shells { inherit (ourPkgs) contrast-releases; };
+        devShells = pkgs.callPackages ./dev-shells { };
 
         formatter = treefmtEval.config.build.wrapper;
 
         checks.formatting = treefmtEval.config.build.check self;
 
-        legacyPackages = pkgs // ourPkgs;
+        legacyPackages = pkgs.contrastPkgs // {
+          nixpkgs = builtins.removeAttrs pkgs [
+            "fenix"
+            "contrastPkgs"
+          ];
+        };
```

Is there something preventing

```suggestion
        legacyPackages =
          pkgs.contrastPkgs
          // (builtins.removeAttrs pkgs [
            "fenix"
            "contrastPkgs"
          ]);
```

from working? Tried it just now, and as far as I can see, both `contrastPkgs` and `pkgs` are available prefix-free

---

in case of a name collision, packages in nixpkgs will silently overwrite our packages.

---

I think I'd like to land this as is, we can see if it is too annoying this way and improve later.